src/Tickets/Commerce/Utils/Currency.php

```diff
@@ -1458,4 +1459,69 @@ public function get_currency_code_options() {
 		 */
 		return apply_filters( 'tec_tickets_commerce_currency_code_options', $options );
 	}
+
+	/**
+	 * Get unsupported currencies and notice texts.
+	 *
+	 * @since TBD
+	 *
+	 * @return array
+	 */
+	public static function get_unsupported_currencies(): array {
+		/**
+		 * Filter all unsupported currenices before returning.
+		 *
+		 * @since TBD
+		 *
+		 * @return string
+		 */
+		return apply_filters( 'tec_tickets_commerce_unsupported_currencies', [
+				'HRK' => [
+					'heading'   => __( 'Tickets Commerce is now selling with Euro', 'event-tickets' ),
+					'message'   => __( 'From the 1st of January 2023, the euro became the official currency for Croatia. We have removed the Croatian kuna from our currency settings and updated your settings to start selling with Euro.', 'event-tickets' ),
+					'new_value' => 'EUR',
+				],
+			]
+		);
+	}
+
+	/**
+	 * Verify if currency is supported.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public static function is_supported_currency(): bool {
```

Comment:
Maybe naming `is_current_currency_supported` will be more self-explanatory, as `is_supported_currency` would be a better name for something that receives a parameter.